Gemfile

```diff
@@ -12,46 +12,21 @@ branch = File.read(File.expand_path("../maintenance-branch", __FILE__)).chomp
   end
 end
 
-if RUBY_VERSION < '1.9.3'
-  gem 'rake', '< 11.0.0' # rake 11 requires Ruby 1.9.3 or later
-elsif RUBY_VERSION < '2.0.0'
-  gem 'rake', '< 12.0.0' # rake 12 requires Ruby 2.0.0 or later
-else
-  gem 'rake', '> 12.3.2'
-end
-
 if ENV['DIFF_LCS_VERSION']
   gem 'diff-lcs', ENV['DIFF_LCS_VERSION']
 else
   gem 'diff-lcs', '~> 1.4', '>= 1.4.3'
 end
 
+gem 'ffi', '> 1.9.24' # prevent Github security vulnerability warning
```

Comment:
Does this not need to be `1.12` due to our lack of rubygems

Reply:
Probably. I tried to keep the changes scoped to the removal of support of Ruby < 2.3
It's the way [it is for Ruby 2.3+ in 4-0-dev](https://github.com/rspec/rspec-mocks/blame/4-0-dev/Gemfile#L44).
I'll be happy to fix it here, and the reason why I didn't is because I find it problematic to foresee what you'd like to keep untouched and what to change.

I don't mind this change, and I clearly understand the problem with ffi 1.13.1 and long-unreleased 1.13.2.

Reply:
I'd make changes like this only to versions known to work..

Reply:
I haven't made any change here 😆 

![image](https://user-images.githubusercontent.com/6916/101817330-47dd7a00-3b33-11eb-87a5-748492c13921.png)

Reply:
`main` is locked to `1.12`